### DISCUSSION TITLE
Pay as you go tier

### DISCUSSION THREAD

**Discussion by Olyno** [supabase/supabase#8243] [Feature Requests]
Hi 👋🏻 
I would personally see a "pay as you go offer". Go from a free tier to 25$/month is too much to me, and i can't afford that when starting my project. There is more people in the same case, that's why i'm suggesting it.

From **Ping for toast** on Discord:

> I hate to be that guy but I was curious if eventually supabase will consider a "hobbiest" tier.
> The free tier is great for local development of products, preparing to launch.
> The 25$ tier is a huge jump. It feels great for actual companies, people launching a service to profit.
> 
> But lots of time, I'm not starting a company. I just want a database for some small service. I just want a few limits knocked off the free tier
> - No automatic shutdown (or even lengthened)
> - Doesn't count to 2 free project quota
> - Maybe a gigabyte included, but even that might be a bit much. Just the opportunity to pay for more storage, say at 0.25/gb
> 
> Services that would be a good fit for this tier would be your weekend projects that you present to the public, but are probably open source and only offered as a good will.
> 
> TD;LR, a tier between "local development" and "company"

**Comment by lesliebong** [comment]
IF I am not mistaken, one of the reasons **Supabase** was built to overcome the unpredictable pricing from the popular Google Firebase services, which is a Pay As You Go offer on their blaze plan.
